src/app/server/Server.cpp

```diff
@@ -333,6 +333,26 @@ void Server::DispatchShutDownAndStopEventLoop()
     PlatformMgr().ScheduleWork(StopEventLoop);
 }
 
+void Server::DispatchSoftwareFaultEvent(uint32_t eventId)
```

Comment:
Why does this make sense as a general API on Server?

Reply:
Just to align with existing implementation such as void DispatchShutDownAndStopEventLoop(), we can move these APIs to cluster server if it make more sense.

Reply:
These APIs shouldn't really exist.  We should have event dispatch APIs, but not these random "happens to call into some random platform manager special case we added for a particular event" APIs....